karrier-furyctl-runner/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM alpine:3.10
 
-LABEL maintainer="niccolo@sighup.io"
+LABEL maintainer="engineering@sighup.io"
 
+ENV ARCH=amd64
+ENV ARCH_ALT=x86_64
 ENV AWSCLI_VERSION 1.19
 ENV GCLOUD_VERSION 344.0.0
 ENV TERRAFORM_VERSION 1.0.1
 ENV KUBECTL_VERSION v1.21.2
 ENV KUSTOMIZE_VERSION v3.6.1
-ENV FURYCTL_VERSION v0.6.1
+ENV FURYCTL_VERSION v0.8.0-rc1
 ENV FURYAGENT_VERSION v0.2.3
 ENV VELERO_VERSION v1.6.0
 ENV DIRENV_VERSION v2.28.0
```

Comment:
Does it worth adding also openpvn binary (better if >= 2.5) ? And update the alpine version to the latest? (that probably also solves the OpenVPN issue had)